test/e2e/metering_manual_install_tests.go

```diff
@@ -113,3 +125,72 @@ func DecodeMeteringConfigManifest(basePath, manifestPath, manifestFilename strin
 
 	return mc, nil
 }
+
+func s3InstallFunc(ctx *deployframework.DeployerCtx) error {
+	// The default ctx.TargetPodsCount value assumes that HDFS
+	// is being used a storage backend, so we need to decrement
+	// that value to ensure we're not going to poll forever waiting
+	// for Pods that will not be created by the metering-ansible-operator
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+	// Before we can create the AWS credentials secret in the ctx.Namespace, we need to ensure
+	// that namespace has been created before attempting to query for a resource that does not exist.
+	_, err := ctx.Client.CoreV1().Namespaces().Get(context.Background(), ctx.Namespace, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
```

Comment:
do you need to handle if the error is non-nil and NOT `IsNotFound`? Same on line 162 (which, if I'm reading correctly, could overwrite this err object since it is shadowing the var).

Reply:
Oh yeah, good catch - looks like we just ignore non-IsNotFound errors.